src/main/java/org/scalameter/annotations/benchmark.java

```diff
@@ -0,0 +1,31 @@
+package org.scalameter.annotations;
```

Comment:
Hm, just one more question from me: what stands for placing the annotation in the `org.scalameter.annotation` and making type aliases for them in `org.scalameter.api` ?